mesonbuild/dependencies/mpi.py

```diff
@@ -27,7 +27,9 @@ def mpi_factory(env: 'Environment',
                 for_machine: 'MachineChoice',
                 kwargs: T.Dict[str, T.Any],
                 methods: T.List[DependencyMethods]) -> T.List['DependencyGenerator']:
-    language = kwargs.get('language', 'c')
+    language = kwargs.get('language')
+    if language is None:
+        language = 'c'
```

Comment:
This should obviously be part of the previous commit.

Reply:
squashed